1.20.2/23w33a.md

```diff
@@ -0,0 +1,3 @@
+debug | The charts on the debug screen can now be toggled with F3+1 (pie chart), F3+2 (FPS and TPS) and F3+3 (bandwidth and ping).
+
+trims | Added `decal` field, defaulting to false. If true, the pattern texture will be masked based on the underlying armor.
```

Comment:
To what? Trim materials? Trim patterns? It should be clear with the text itself, ignoring the tags.